app/assets/javascripts/update-messages.js

```diff
@@ -0,0 +1,62 @@
+function buildHTML(message) {
+　var insertImage = '';
+  var message_image = ""
+  $("html,body").animate({scrollTop:$('.messages')[0].scrollHeight}, 'fast');
```

Comment:
スクロールの記述は.done(function(json)の中の最後あたりに書いた方がわかりやすいと思いました！
(ここに記述してもスクロール動くんですね。。。)

Reply:
` .done(function(json) {
      var id = $('.main__chatcomments').last().data('message-id');
      var insertHTML = '';
      json.messages.forEach(function(message) {
        if (message.id > id ) {
          insertHTML += buildHTML(message);
          $("html,body").animate({scrollTop:$('.messages')[0].scrollHeight}, 'fast');
        }
      });
      $('.messages').append(insertHTML);
    })`

自動スクロールの記述はdoneメソッドの中で、コメントが追加されたときに作動するようにしました！
```suggestion
  $("html,body").animate({scrollTop:$('.messages')[0].scrollHeight}, 'fast');
```